components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/listener/AccountLockTenantMgtListener.java

```diff
@@ -39,10 +41,16 @@ public void onTenantInitialActivation(int tenantId) throws StratosException {
         if (log.isDebugEnabled()) {
             log.debug("AccountLockTenantMgtListener is fired for Tenant ID : " + tenantId);
         }
-
+        String tenantDomain = IdentityTenantUtil.getTenantDomain(tenantId);
+        boolean isOrganization = false;
```

Comment:
why this is initiallized outside the try block.
I see the usage only inside the try block

Reply:
Resolved by https://github.com/wso2-extensions/identity-event-handler-account-lock/pull/136/commits/7451b33257f11bbf21e569335f62b3e287123479